MAUI/ImageEditor/save.md

```diff
@@ -213,4 +213,108 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
 
 {% endtabs %}
 
+## Reset
```

Comment:
split Reset in separate md file

Reset image using .NET MAUI Image Editor (SfImageEditor)

Reply:
Added a separate file for reset